pip/req/req_set.py

```diff
@@ -281,18 +279,14 @@ def prepare_files(self, finder):
                 is_wheel = False
                 if req_to_install.editable:
                     if req_to_install.source_dir is None:
```

Comment:
It's hard for me to understand exactly what's happening from the diff, but I imagine this was some simple moving around of stuff that you're confident about.